src/Service/DashboardHelper.php

```diff
@@ -150,11 +159,20 @@ private function getBoardColumnsCaseInformation(Municipality $municipality): arr
                 'count' => $this->caseRepository->findCountOfCasesWithSomeExceededDeadlineBy(['board' => $board]),
             ];
 
-            $boardCount = array_sum(array_column($rows, 'count'));
+            // TODO: When all rows are fully implemented update beneath
+//            $boardCount = array_sum(array_column($rows, 'count'));
```

Comment:
Remove this line (and maybe the “TODO” above).

Reply:
Reverted this change in c22aeb68fb47a45b67c0be2f7a44b145efa208dc and hid unimplemented rows in dashboard grid instead.